pillar/heroku/micromasters.sls

```diff
@@ -0,0 +1,212 @@
+    GOOGLE_API_KEY: __vault__::secret-{{ business_unit }}/google>data>api_key
+    GTM_CONTAINER_ID: {{ env_data.GTM_CONTAINER_ID }}
+    MAILGUN_FROM_EMAIL: {{ env_data.MAILGUN_FROM_EMAIL }}
+    MAILGUN_KEY: __vault__::secret-operations/mailgun>data>api_key
+    MAILGUN_URL: {{ env_data.MAILGUN_URL }}
+    MICROMASTERS_BASE_URL: {{ env_data.MICROMASTERS_BASE_URL }}
+    MICROMASTERS_CORS_ORIGIN_WHITELIST: {{ env_data.MICROMASTERS_CORS_ORIGIN_WHITELIST }}
+    MICROMASTERS_DB_CONN_MAX_AGE: 0
+    MICROMASTERS_DB_DISABLE_SSL: True
+    MICROMASTERS_EMAIL_HOST: __vault__::secret-operations/mit-smtp>data>relay_host
+    MICROMASTERS_EMAIL_PASSWORD: __vault__::secret-operations/mit-smtp>data>relay_password
+    MICROMASTERS_EMAIL_PORT: 587
+    MICROMASTERS_EMAIL_TLS: True
+    MICROMASTERS_EMAIL_USER: __vault__::secret-operations/mit-smtp>data>relay_username
+    MICROMASTERS_ENVIRONMENT: {{ env_data.env_name }}
+    MICROMASTERS_FROM_EMAIL: 'MITx MicroMasters <micromasters-support@mit.edu>'
+    MICROMASTERS_LOG_LEVEL: {{ env_data.MICROMASTERS_LOG_LEVEL }}
+    MICROMASTERS_SUPPORT_EMAIL: 'micromasters-support@mit.edu'
+    MICROMASTERS_USE_S3: True
+    NODE_MODULES_CACHE: False
+    OPEN_DISCUSSIONS_API_USERNAME: {{ env_data.OPEN_DISCUSSIONS_API_USERNAME }}
+    OPEN_DISCUSSIONS_BASE_URL: {{ env_data.OPEN_DISCUSSIONS_BASE_URL }}
+    OPEN_DISCUSSIONS_COOKIE_DOMAIN: {{ env_data.OPEN_DISCUSSIONS_COOKIE_DOMAIN }}
+    OPEN_DISCUSSIONS_COOKIE_NAME: {{ env_data.OPEN_DISCUSSIONS_COOKIE_NAME }}
+    OPEN_DISCUSSIONS_JWT_SECRET: __vault__::secret-{{ business_unit }}/open-discussions>data>jwt_secret
+    OPEN_DISCUSSIONS_REDIRECT_COMPLETE_URL: '/complete/micromasters'
+    OPEN_DISCUSSIONS_REDIRECT_URL: {{ env_data.OPEN_DISCUSSIONS_BASE_URL }}
+    OPEN_DISCUSSIONS_SITE_KEY: 'micromasters'
+    OPEN_EXCHANGE_RATES_APP_ID: __vault__::secret-{{ business_unit }}/open-exchange-rates>data>app_id
+    OPEN_EXCHANGE_RATES_URL: 'https://openexchangerates.org/api/'
+    OPENSEARCH_HTTP_AUTH: __vault__::secret-{{ business_unit }}/opensearch>data>http_auth
+    OPENSEARCH_INDEX: 'micromasters'
+    OPENSEARCH_URL: __vault__::secret-{{ business_unit }}/opensearch>data>url
+    PGBOUNCER_DEFAULT_POOL_SIZE: {{ env_data.PGBOUNCER_DEFAULT_POOL_SIZE }}
+    PGBOUNCER_MAX_CLIENT_CONN: {{ env_data.PGBOUNCER_MAX_CLIENT_CONN }}
+    SECRET_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/django>data>secret_key    
+    SENTRY_AUTH_TOKEN: __vault__::secret-{{ business_unit }}/sentry>data>auth_token                 
+    SENTRY_DSN: __vault__::secret-{{ business_unit }}/sentry>data>dsn
+    SENTRY_ORG_NAME: 'mit-office-of-digital-learning'
+    SENTRY_PROJECT_NAME: 'micromasters'
+    STATUS_TOKEN:  __vault__:gen_if_missing:secret-{{ business_unit }}/django>data>status_token                     
+    UWSGI_PROCESS_COUNT: 4
+    UWSGI_SOCKET_TIMEOUT: 1                      
+    UWSGI_THREAD_COUNT: 50
+    {% if env_data.env_name == 'production' %}
```

Comment:
Should this get updated to be conditional on RC and production, now that we have the additional stacks in Pulumi?